.github/workflows/deploy-preview.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         shell: bash
         run: |
           mkdir -p ${PWD}/dist
-          docker run -v ${PWD}/dist:/hugo/dist -v "${PWD}/${{ inputs.source_directory }}:/hugo/${{ inputs.website_directory }}" --rm grafana/docs-base:latest /bin/bash -c 'HUGO_SSI=false hugo --environment=docs --destination=dist/ --baseURL= --minify'
+          docker run -v ${PWD}/dist:/hugo/dist -v "${PWD}/${{ inputs.source_directory }}:/hugo/${{ inputs.website_directory }}" --rm grafana/docs-base:latest /bin/bash -c 'echo -e "---\\nredirectURL: \"/docs/grafana/latest\"\\ntype: redirect\\nversioned: true\\n---\\n" > "content/docs/grafana/_index.md" && HUGO_SSI=false hugo --environment=docs --destination=dist/ --baseURL= --minify'
```

Comment:
Think this should fix for Grafana but we'll need to parameterize the project to make this generic for any repo.